geoips/commandline/run_procflow.py

```diff
@@ -32,61 +32,68 @@ def main(get_command_line_args_func=None):
     """
     DATETIMES = {}
     DATETIMES["start"] = datetime.utcnow()
-    LOG = setup_logging()
 
     if get_command_line_args_func is None:
         get_command_line_args_func = get_command_line_args
-    LOG.info("GETTING COMMAND LINE ARGUMENTS")
     # arglist=None allows all possible arguments.
     ARGS = get_command_line_args_func(
         arglist=None, description="Run data file processing"
     )
 
+    COMMAND_LINE_ARGS = ARGS.__dict__
```

Comment:
Note this is the only code change required for interactive log setup in the files with lots and lots of interactive logs added.  Every other modified file is JUST the addition of LOG.interactive commands.

So, run_procflow.py, args.py, and log_setup.py were the only files modified for adding the interactive log level.  run_procflow.py plus ALL the other modified files were just adding interactive logs, or removing print statements / shell output.